corehq/apps/reports/static/reports/js/bootstrap3/case_details.js

```diff
@@ -72,20 +72,13 @@ hqDefine("reports/js/bootstrap3/case_details", [
 
         self.data_loading = ko.observable(false);
 
-        self.getParameterByName = function (name, url) {
```

Comment:
This is only ever called with `('form_id', window.location.hash)`, so I've in-lined those arguments.